rest/route/github.go

```diff
@@ -103,16 +103,17 @@ func (gh *githubHookApi) Parse(ctx context.Context, r *http.Request) error {
 
 // shouldSkipWebhook returns true if the event is from a GitHub app and the app is not set up or,
```

Comment:
would update this comment for the new behavior (the app is available for the owner/repo)